simpledotorg/simple-server
# Update support number in Sri Lanka

**Story card:** [sc-12162](https://app.shortcut.com/simpledotorg/story/12162/update-help-number-in-sri-lanka)

## Because

Request to update the support phone number in Sri Lanka

## This addresses

Updates the corresponding env variable.

## Test instructions

- DEFAULT_COUNTRY=LK rails s
- Visit http://localhost:3000/api/v3/help.html
- The page should show the new phone number, and click the link should open the appropriate whatsapp deeplink.
